py-polars/tests/unit/ml/test_to_jax.py

```diff
@@ -0,0 +1,156 @@
+from __future__ import annotations
+
+import sys
+from typing import TYPE_CHECKING, Any
+
+import pytest
+
+import polars as pl
+import polars.selectors as cs
+from polars.dependencies import _lazy_import
+
+# don't import jax until an actual test is triggered (the decorator already
+# ensures the tests aren't run locally; this avoids premature local import)
+jx, _ = _lazy_import("jax")
+jxn, _ = _lazy_import("jax.numpy")
+
+if TYPE_CHECKING:
+    from polars.datatypes import PolarsDataType
+
+
+@pytest.fixture()
+def df() -> pl.DataFrame:
+    return pl.DataFrame(
+        {
+            "x": [1, 2, 2, 3],
+            "y": [1, 0, 1, 0],
+            "z": [1.5, -0.5, 0.0, -2.0],
+        },
+        schema_overrides={"x": pl.Int8, "z": pl.Float32},
+    )
+
+
+@pytest.mark.ci_only()
+class TestJaxIntegration:
```

Comment:
Why are these in a class? You can just use `pytestmark = pytest.mark.ci_only` if you want to mark all those tests as ci_only.

Reply:
Mostly because I didn't know you could do that, heh :))
(We currently use a different test harness at work)

Updated - and did the same for the torch tests 👍